ssl/ssl_lib.c

```diff
@@ -4944,3 +4957,136 @@ uint32_t SSL_get_max_early_data(const SSL *s)
 {
     return s->max_early_data;
 }
+
+SSL_SESSION_CACHE *SSL_SESSION_CACHE_new(
+    unsigned long (*hash)(const SSL_SESSION *),
+    int (*cmp)(const SSL_SESSION *, const SSL_SESSION *))
+{
+    SSL_SESSION_CACHE *ret;
```

Comment:
blank line after decl ... 
You could merge declaration with his next assignment line too .